using-stm/src/test/java/org/acme/quickstart/stm/STMResourceTest.java

```diff
@@ -0,0 +1,42 @@
+package org.acme.quickstart.stm;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import io.restassured.RestAssured;
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+class STMResourceTest {
```

Comment:
Can we have the @SubstrateTest equivalent?

Reply:
+1